plugins/speculation-rules/helper.php

```diff
@@ -87,6 +87,12 @@ function plsr_get_speculation_rules() {
 							'href_matches' => $href_exclude_paths,
 						),
 					),
+					// Also exclude rel=nofollow links, as plugins like WooCommerce attribute their add-to-cart links.
```

Comment:
This reads weird to me.

```suggestion
					// Also exclude rel=nofollow links, as plugins like WooCommerce use that on their add-to-cart links.
```

Reply:
I was trying to be clever and use "attribute" as a verb. But clearly my pun was a dud.